tools/http3_test/src/lib.rs

```diff
@@ -467,7 +467,7 @@ impl Http3Test {
 
             info!("sending HTTP request {:?}", req.hdrs);
 
-            let s =
+            let (s, _hdrs_fully_sent) =
```

Comment:
We want to kill http3_test anyway, so I don't see much point perfecting this client